e2e/tests/functional/planning/timelist.e2e.spec.js

```diff
@@ -22,6 +22,18 @@
 
 import { createDomainObjectWithDefaults, createPlanFromJSON } from '../../../appActions.js';
 import { expect, test } from '../../../pluginFixtures.js';
+const { getEarliestStartTime } = require('../../../helper/planningUtils');
+const examplePlanSmall3 = require('../../../test-data/examplePlans/ExamplePlan_Small3.json');
```

Comment:
We're all ESM now, so we need to update these imports:

```suggestion
import { getEarliestStartTime } from '../../../helper/planningUtils.js';
const examplePlanSmall = JSON.parse(
  fs.readFileSync(new URL('../../test-data/examplePlans/ExamplePlan_Small3.json', import.meta.url))
);
```